apps/server/src/shared/repo/importuser/importuser.repo.integration.spec.ts

```diff
@@ -750,7 +752,7 @@ describe('ImportUserRepo', () => {
 				await Promise.all(result);
 			});
 			it('[UNIQUE] should prohibit same match of one user ', async () => {
-				await em.getDriver().ensureIndexes();
+				await orm.getSchemaGenerator().ensureIndexes();
```

Comment:
What is the different between em.getDriver() and orm.getSchemaGenerator()?

Reply:
em.getDriver().ensureIndexes(); is deprecated in MikroOrm 5.x